tests/unit/examples-spec.js

```diff
@@ -36,4 +41,10 @@ describe('examples', function () {
             port: 2222
         }]);
     });
+
+    it('same name sub-property as parent property', function () {
```

Comment:
Should also check that winstonLogger property is left as is?

Reply:
Done above in `sails-local` config